src/rez/config.py

```diff
@@ -975,3 +1050,9 @@ def get_module_root_config():
 
 # singleton
 config = Config._create_main_config()
+
+if os.getenv("REZ_LOG_DEPRECATION_WARNINGS"):
```

Comment:
We should add this to the changelog to give a heads up that config files will be fored-loaded.

Reply:
@instinct-vfx I added a warning in both the docs and the changelog.